src/shorthands/JImage.jl

```diff
@@ -0,0 +1,19 @@
+function _JImage(pos, img, centering, shapeargs, shape, scaleargs)
+    if shape != false
+        shape(shapeargs...)
+    end
+    scale(scaleargs)
+    placeimage(img, pos, centered = centering)
+    return pos
+end
+
+JImage(pos::Point, img, centering = true; shapeargs = (), shape = false, scaleargs = 1) =
```

Comment:
I think `shape = nothing` would be a better default.

Reply:
The function needs a docstring which should probably explain clipping as part of shapeargs.

Reply:
That makes sense. Done.